sys/kern/tmpfs.c

```diff
@@ -945,9 +992,38 @@ static int tmpfs_resize(tmpfs_mount_t *tfm, tmpfs_node_t *v, size_t newsize) {
   }
 
   v->tfn_size = newsize;
+  tmpfs_update_time(v, TMPFS_UPDATE_CTIME | TMPFS_UPDATE_MTIME);
   return 0;
 }
 
+static int tmpfs_chtimes(tmpfs_node_t *v, timespec_t *atime, timespec_t *mtime,
```

Comment:
Why don't you update ctime in this function?

Reply:
I'm not really sure what you are asking about, because there are two things. The first is that the ctime cannot be changed with this interface, but the second one is that you're right I forgot to update `ctime`.

Reply:
The second think is what I mean.